apps/wallet/configs/webpack/webpack.config.common.ts

```diff
@@ -177,6 +177,9 @@ const commonConfig: () => Promise<Configuration> = async () => {
                 // TODO: check if this is worth investigating a fix and maybe do a separate build for UI and bg?
                 // 'typeof window': JSON.stringify(typeof {}),
                 'process.env.NODE_DEBUG': false,
+                'process.env.WALLET_VERSION': JSON.stringify(
```

Comment:
Can you just import the package.json and read the version field instead of injecting through the process like this?